source/fem-dem/parameters_cfd_dem.cc

```diff
@@ -45,6 +45,16 @@ namespace Parameters
       "0",
       Patterns::Double(),
       "The refinement factor used to calculate the number of pseudo-particles in the satellite point method");
+    prm.declare_entry(
+      "qcm sphere diameter",
+      "0",
+      Patterns::Double(),
+      "The diameter of the reference sphere for QCM void fraction scheme");
+    prm.declare_entry(
+      "qcm sphere equal cell volume",
+      "false",
+      Patterns::Bool(),
+      "Specify whether the virtual sphere has same volume as element");
```

Comment:
So the default behavior is to use the cell size as the radius is that is?

Reply:
yes